project_setup/Skeletonize-existing-repository.md

```diff
@@ -0,0 +1,84 @@
+# Skeletonize an existing repository #
+
+Skeletonizing a repository standardizes our development setup and environment
+as well as enables our [Lineage](https://github.com/cisagov/action-lineage/)
+Github Action to keep it updated and standardized.
+
+## About skeletons ##
+
+Skeleton projects contain [licensing information](LICENSE), as
+well as [pre-commit hooks](https://pre-commit.com) and
+[GitHub Actions](https://github.com/features/actions) configurations
+appropriate for the major languages that we use. This lets us standardize
+[cisagov](https://github.com/cisagov) GitHub projects to a
+[list of cisagov skeleton projects](https://github.com/search?q=org%3Acisagov+skeleton&type=Repositories).
+
+## General overview ##
+
+The general outline of how to add a skeleton to a repository is:
+
+1. Add the skeleton as a remote to the non-skeletonized repository
+2. Pull with `--allow-unrelated-histories`
+3. Fix all the inevitable conflicts (this takes the longest)
+4. Run `pre-commit install`
+5. Fix problems that may arise from `pre-commit` being added
+6. Commit to a branch and submit a PR
```

Comment:
Is it possible to add hyperlinks to the appropriate sections of this document to (at least some of) these enumerated values?

Reply:
That's a great idea, and it's definitely possible.

Reply:
Added! https://github.com/cisagov/development-guide/pull/32/commits/d062c46acca321119bd4f4998b5a2652948fb345